cdap-runtime-ext-dataproc/src/main/resources/gce-dataproc.json

```diff
@@ -220,6 +220,47 @@
           }
         }
       ]
+    },
+    {
+      "label": "Polling Settings",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Create Poll Delay",
+          "name": "pollCreateDelay",
+          "description": "The number of seconds to wait after creating a cluster to begin polling to see if it has been created",
+          "widget-attributes": {
+            "default": "60"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Create Poll Jitter",
+          "name": "pollCreateJitter",
+          "description": "Maximum amount of jitter in seconds to add to the delay for polling for cluster creation",
```

Comment:
Why do we need a jitter, and not just a `pollCreateDelay`?

Reply:
this is an attempt to combat the scenario where a bunch of pipelines start at the same time and try to call the APIs at the same time.